spec/spec_helper.rb

```diff
@@ -26,8 +33,14 @@
   end
 
   config.after(:each) do
-    # ActiveRecord::Base.connection.tables does not include partitioned tables in Rails 5.1
-    ActiveRecord::Base.connection.select_values("SELECT tablename FROM pg_tables WHERE schemaname = 'public'").each do |table|
+    ActiveRecord::Base.connection.execute("DROP EXTENSION IF EXISTS pg_partman CASCADE")
```

Comment:
Earlier I'd noted we should probably bump this to a PR that needs it, but when we do that it strikes me that we should probably also extract this code so that we don't have to maintain it two different places in the same file.

Reply:
Yup that's probably a good idea. We'll include that as a separate commit in the next partman PR